### ISSUE TITLE
GMFCS level V

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7282]
**Preferred term label:**
GMFCS level V

**Synonyms**
The Gross Motor Function Classification System level V

**Definition (free text, please give PubMed ID)**
Children with cerebral palsy classified under GMFCS level V are transported in a manual wheelchair in all settings; they are limited in their ability to maintain antigravity head and trunk postures and control leg and arm movements (PMID: 27188686).

**Parent term (use hpo.jax.org/app)**
Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
As mentioned, I think these terms are not good for HPO because they bundle a number of different items. For HPO, we want to capture the detailed data.
